appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -122,6 +122,7 @@ class CreateUserOnBuddyRequestTest extends AbstractAppServiceIntegrationTest
 		def newNickname = "Bobby"
 		def newPassword = "B o b"
 		def updatedBobJson = bob.convertToJson()
+		updatedBobJson.yonaPassword = newPassword
```

Comment:
With update user on buddy request, previously the Yona-Password header was set to the new password and the overwrite user confirmation code was sent. Now both the JSON yonaPassword property and the Yona-Password header are sent. However, the first is ignored, isn't it? It might be good to document that or do a validation that both are the same?